### ISSUE TITLE
Support parameters such as ntree_limit when predict in xgboost

### ISSUE THREAD

**Issue by xxxxsk** [mars-project/mars#1965]
In my scene, I get a model file in xgboost format, I need to deploy the model and predict some score. While I need to set ntree_limit when predict but I got TypeError:
```
TypeError: predict got an unexpected keyword argument 'ntree_limit'
```
Below is the example I provided:
```python
import pandas as pd 
from xgboost import XGBClassifier
from mars.learn.contrib import xgboost as xgb
from sklearn import datasets
    
dataset = datasets.load_breast_cancer()
y = dataset['target']
x = pd.DataFrame(dataset['data'],columns=dataset['feature_names'])

model = XGBClassifier()
model.fit(x,y)
print(model.predict(x[:10],ntree_limit=30))
model.save_model('mars_try.model')

mars_model = xgb.XGBClassifier()
mars_model.load_model('mars_try.model')
mars_model.predict(x[:10],ntree_limit=30)
```
Is it possible to parameters such as ntree_limit when predict in xgboost in Mars?

**Comment by qinxuye** [comment]
Yeah, I think it's not hard to add the argument, we will support it ASAP.

**Comment by xxxxsk** [comment]
> Yeah, I think it's not hard to add the argument, we will support it ASAP.

thanks！

**Comment by qinxuye** [comment]
Since I am fixing some issue in XGBoost(#1966), I added the support for extra arguments in `predict` without an effort, this PR will be back ported to v0.6 branch, and will be included in v0.7.0a6 as well as v0.6.5 .
